### PR TITLE
Feature/notifications + ProfileAvatar / TopBar minor improvements

### PR DESCRIPTION
**Pull request by aslakihle**

### CODE REVIEW

File: src/components/DataDisplay/ProfileAvatar.tsx [hunk 1]
```diff
@@ -61,6 +65,42 @@ const Avatar = styled(EDSAvatar)`
   margin-right: ${spacings.comfortable.medium};
```
Comment (wigsnes) [inline src/components/DataDisplay/ProfileAvatar.tsx:65]: Should remove this

File: src/components/DataDisplay/ProfileAvatar.tsx [hunk 2]
```diff
@@ -61,6 +64,41 @@ const Avatar = styled(EDSAvatar)`
...
 `;
 
+const getFirstCharacterAfterComma = (name: string) => {
+  const nameSplitOnComma = name.split(',');
+  return nameSplitOnComma[1].trim().charAt(0);
+};
+
+export const createInitialsFromName = (name: string | undefined) => {
+  const defaultName = 'XX';
```
Comment (wigsnes) [inline src/components/DataDisplay/ProfileAvatar.tsx:73]: Er det bestemt at XX skal være default?

File: src/components/DataDisplay/ProfileAvatar.tsx [hunk 3]
```diff
@@ -61,6 +64,41 @@ const Avatar = styled(EDSAvatar)`
...
+  if (!name) return defaultName;
+
+  const nameWithoutParenthesis = name
+    .replace(/ *\([^)]*\) */g, '')
+    .toUpperCase();
+  const lastNameCommaFirstName = nameWithoutParenthesis.includes(',');
```
Comment (wigsnes) [inline src/components/DataDisplay/ProfileAvatar.tsx:79]: You should move this down right before it is used

File: src/components/DataDisplay/ProfileAvatar.tsx [hunk 4]
```diff
@@ -61,6 +64,41 @@ const Avatar = styled(EDSAvatar)`
...
+  const splitNames = nameWithoutParenthesis.split(' ');
+
+  if (splitNames.length === 1 && splitNames[0] !== '') {
+    return splitNames[0].charAt(0) + '.';
```
Comment (wigsnes) [inline src/components/DataDisplay/ProfileAvatar.tsx:83]: Also here, did the designers agree on a period if user only have one name?

File: src/components/DataDisplay/ProfileAvatar.tsx [hunk 5]
```diff
@@ -61,6 +64,41 @@ const Avatar = styled(EDSAvatar)`
...
+
+  if (lastNameCommaFirstName) {
+    return (
+      getFirstCharacterAfterComma(nameWithoutParenthesis) +
+      splitNames[0].charAt(0)
+    );
+  }
+  if (splitNames.length >= 2) {
+    return (
+      splitNames[0].charAt(0).toUpperCase() +
```
Comment (wigsnes) [inline src/components/DataDisplay/ProfileAvatar.tsx:94]: You probably dont need .toUppercase here?

**Review by mariush2 — CHANGES_REQUESTED** [review]

**Pull request merged**